src/main/java/teammates/common/datatransfer/FeedbackQuestionDetails.java

```diff
@@ -182,7 +177,7 @@ public String getNoResponseTextInHtml(String giverEmail, String recipientEmail,
                + "</i>";
     }
 
-    public boolean shouldShowNoResponseText(String giverEmail, String recipientEmail,
+    public boolean shouldShowNoResponseText(String recipientEmail,
```

Comment:
The `recipientEmail` param doesn't look used either. See if you can remove it as well.